ocs_ci/ocs/cluster.py

```diff
@@ -884,15 +915,8 @@ def validate_cluster_on_pvc():
         pvc_names,
         constants.CEPH_ROOK_IO_PVC_LABEL,
     )
-    osd_pods = get_pod_name_by_pattern("rook-ceph-osd", ns, filter="prepare")
-    for ceph_pod in mon_pods + osd_pods:
-        out = run_cmd(f"oc -n {ns} get pods {ceph_pod} -o yaml")
-        out_yaml = yaml.safe_load(out)
-        for vol in out_yaml["spec"]["volumes"]:
-            if vol.get("persistentVolumeClaim"):
-                claimName = vol.get("persistentVolumeClaim").get("claimName")
-                logger.info(f"{ceph_pod} backed by pvc {claimName}")
-                assert claimName in pvc_names, "Ceph Internal Volume not backed by PVC"
+    validated_pods = set()
+    validate_claim_name_match_pvc(pvc_names, validated_pods)
```

Comment:
Since we default validated_pods to None and create a set in the function if it is None, can we skip initializing it here and passing through an empty set?

```suggestion
    validate_claim_name_match_pvc(pvc_names)
```

Reply:
I explained in comment above, as I would like to speed up process in next re-try I need to use the set from outside of the function.